taskfile/read/taskfile.go

```diff
@@ -170,3 +179,25 @@ func exists(path string) (string, error) {
 
 	return "", fmt.Errorf(`task: No Taskfile found in "%s". Use "task --init" to create a new one`, path)
 }
+
+func checkCircularIncludes(node *ReaderNode) error {
+	if node == nil {
+		return errors.New("failed to check for include cycle: node was nil")
```

Comment:
We prefix all errors with `task: `.

```suggestion
		return errors.New("task: failed to check for include cycle: node was nil")
```